### PR TITLE
docs: render and key views

### LINKED ISSUE #38: docs: keying views
We have made changes to how to key views in ontime

These is currently explained in the OBS section.
We would need to update the documentation to mention the variable and the behaviour with the key property

We could also consider generalising the page rendering Ontime in a 3rd party software (doesnt quite roll out of the tongue)
https://github.com/cpvalente/ontime/issues/976

### PR DESCRIPTION
**Pull request by cpvalente**
Improving documentation for keying views

@lukestein , what do you think?
Would you be able to provide a screenshot of an Ontime view being rendered in another software (like OBS)

And btw, can you see the previews in vercel?

Closes #38

### CODE REVIEW

**Comment by lukestein** [comment]
@cpvalente: I just signed up for a Vercel account (never used it before) and I think this means it's waiting for your approval, not mine?
image

**Comment by cpvalente** [comment]
> @cpvalente: I just signed up for a Vercel account (never used it before) and I think this means it's waiting for your approval, not mine?
> 
> image

Sorry about this, I did not know you would need an account

I have approved the request, hopefully that means you can see all previews, not just this one

File: src/content/docs/interface/public/lower-thirds.mdx [hunk 1]
```diff
@@ -8,6 +8,6 @@ https://MY-IP-ADDRESS:4001/lower
 ```
 
 The `lower thirds` page is an automated view which generates lower thirds based on current runtime status. \
-This workflow allows for integration with video servers capable of rendering a webpage, such as [OBS Studio](/quick-tips/lower-thirds-in-obs).
+This workflow allows for integration with video servers capable of rendering a webpage, such as [OBS Studio](/quick-tips/render-elsewhere).
```
Comment (lukestein) [inline src/content/docs/interface/public/lower-thirds.mdx:11]:
```suggestion
This workflow allows for [integration with video servers capable of rendering a webpage](/quick-tips/render-elsewhere), such as OBS Studio.
```

File: src/content/docs/quick-tips/render-elsewhere.mdx [hunk 2]
```diff
@@ -0,0 +1,26 @@
+---
+title: Render views in third party apps
+description: Render Ontime views in third party apps
+---
+
+It is possible to render the any view of Ontime in a third party software that is currently networked with Ontime. \
+This flow would be in any software which can render a HTML page, such as OBS (Open Broadcaster Software), disguise or Touchdesigner.
```
Comment (lukestein) [inline src/content/docs/quick-tips/render-elsewhere.mdx:7]:
```suggestion
It is possible to render Ontime output views on any computer that can access Ontime over the network. \
Aside from a web browser, you can do this using any third-party software that can render an HTML page, such as OBS (Open Broadcaster Software), disguise or Touchdesigner.
```

File: src/content/docs/quick-tips/render-elsewhere.mdx [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+:::note[Keying views]
```
Comment (lukestein) [inline src/content/docs/quick-tips/render-elsewhere.mdx:9]:
```suggestion
:::note[Background colors, keying, and transparency]
```

File: src/content/docs/quick-tips/render-elsewhere.mdx [hunk 4]
```diff
@@ -0,0 +1,26 @@
...
+In some views, you will have a key parameter which allows you to configure the background colour.\
+See [view configuration](/quick-tips/configuring-views).
+
+In all other cases you can make use of the [CSS variables](/features/custom-styling) which provide overrides for the background colour:
```
Comment (lukestein) [inline src/content/docs/quick-tips/render-elsewhere.mdx:13]:
```suggestion
Several Ontime views (e.g., the Minimal Timer and Lower Third views) offer a customizable background colour—including transparency—via the `key` parameter.\
See [view configuration](/quick-tips/configuring-views).

For views that do not explicitly offer a customizable `key` color, you can customize the background (or other) colors by overriding [CSS variables](/features/custom-styling) using code such as:
```

File: src/content/docs/quick-tips/render-elsewhere.mdx [hunk 5]
```diff
@@ -0,0 +1,26 @@
...
+:root {
+    --background-color-override: #00FF00;
+}
+```
+:::
+
+## Rendering lower thirds in OBS
+![Lower thirds in OBS](../../../assets/screenshots/features/obs-lower-thirds.png)
+
+* The options **Shutdown source when not visible** and **Refresh browser when scene becomes active** ensure that the lower third animation play on load / unload
```
Comment (lukestein) [inline src/content/docs/quick-tips/render-elsewhere.mdx:25]: Does any animation actually play on unload? I'd have guessed that shutting down the source just queues up the animation on next load, so
> The options **Shutdown source when not visible** and **Refresh browser when scene becomes active** ensure that the lower third animation plays each time the overlay is reloaded

File: src/content/docs/quick-tips/render-elsewhere.mdx [hunk 6]
```diff
@@ -0,0 +1,26 @@
...
+* Remember to clear the CSS to ensure a keyable background (see above)
```
Comment (lukestein) [inline src/content/docs/quick-tips/render-elsewhere.mdx:26]:
```suggestion
* Remember to set a keyable / transparent background color using the `key` parameter or custom CSS (see above)
```

File: src/content/docs/quick-tips/render-elsewhere.mdx (hunk 5 above)
Comment (cpvalente) [reply src/content/docs/quick-tips/render-elsewhere.mdx:25]: I will need to double-check this. I am not usually an OBS user and this documentation is old.
But yes, the animation should unload and OBS will shutdown the source when the frame is clear

**Comment by lukestein** [comment]
Suggestion for L3 page

image

**Pull request merged**